pkg/azure/auth_test.go

```diff
@@ -0,0 +1,62 @@
+package azure
+
+import (
+	"os"
+	"time"
+
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/klog/v2"
+)
+
+var _ = Describe("Auth Tests", func() {
+	When("test getAuthorizer fails", func() {
+		BeforeEach(func() {
```

Comment:
Can you use the `Skip` instead of commenting.
https://onsi.github.io/ginkgo/#skipping-specs

Reply:
yeah that makes sense. Fixed.